django_netjsongraph/base/link.py

```diff
@@ -0,0 +1,89 @@
+import json
+from collections import OrderedDict
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.db.models import Q
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+from jsonfield import JSONField
+from model_utils import Choices
+from model_utils.fields import StatusField
+from rest_framework.utils.encoders import JSONEncoder
+
+from .base import TimeStampedEditableModel
+
+
+@python_2_unicode_compatible
+class BaseLink(TimeStampedEditableModel):
```

Comment:
Let's follow the same convention that we are using in other OpenWISP/NetJSON packages: use the Abstract prefix, eg: `AbstractLink`, applies for the other models as well